### PR TITLE
Add controller tests

### PR DESCRIPTION
**Pull request by AKeeganDev**
This pull request features 3 tests for my user and posts controllers. The three tests each check

- That the index is rendered properly
- Placeholder text is properly rendered
- response code 200 is sent when calling the correct paths

### CODE REVIEW

File: spec/controllers/posts_controller_spec.rb [hunk 1]
```diff
@@ -4,12 +4,17 @@
...
-  # end
+  it 'renders the index template' do
+    get user_posts_path(2)
+    expect(response).to render_template('index')
+  end
+
+  it 'placeholder text test' do
+    get user_posts_path(2)
+    expect(response.body).to include('posts/index')
+  end
```
Comment (andersonlebon) [inline spec/controllers/posts_controller_spec.rb]: - [x] Kindly add a test for the user/show action. hint :point_down: 

```rb
expect(response).to render_template(:show)
```

File: spec/controllers/users_controller_spec.rb [hunk 2]
```diff
@@ -8,8 +8,13 @@
...
   end
 
-  # it 'renders the index template' do
-  #   get users_path
-  #   expect
-  # end
+  it 'renders the index template' do
+    get users_path
+    expect(response).to render_template('index')
+  end
```
Comment (andersonlebon) [inline spec/controllers/users_controller_spec.rb:14]: - [x] Kindly make sure the user/show action is tested. Hint :point_down: 
```rb
expect(response).to render_template(:show)
```

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @AKeeganDev 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
## Hi @AKeeganDev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**